vars/IntTestsPipeline.groovy

```diff
@@ -0,0 +1,143 @@
+
+def call(){
+    Csc csc = new Csc()
+    properties(
+        [
+        buildDiscarder
+            (logRotator (
+                artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '',
+                daysToKeepStr: '14',
+                numToKeepStr: '10'
+            ) ),
+        disableConcurrentBuilds()
+        ]
+    )
+    pipeline {
+        agent {
+            docker {
+                image 'ts-dockerhub.lsst.org/conda_package_builder:latest'
+                alwaysPull true
+                label 'CSC_Conda_Node'
+                args "--env LSST_DDS_DOMAIN=citest --env TS_XML_VERSION=${params.XML_Version} --env TS_SAL_VERSION=${params.SAL_Version} -u root --entrypoint=''"
+                registryUrl 'https://ts-dockerhub.lsst.org'
+                registryCredentialsId 'nexus3-lsst_jenkins'
+            }
+        }
+        options {
+            disableConcurrentBuilds()
+        }
+        environment {
+            PYPI_CREDS = credentials("pypi")
+        }
+        parameters {
+            booleanParam(defaultValue: false, description: "Is this a development build?", name: 'develop')
+        }
+        stages {
+            stage("Create Conda Package") {
+                when {
+                    buildingTag()
+                }
+                steps {
+                    withEnv(["WHOME=${env.WORKSPACE}", "TS_SAL_VERSION=${env.TS_SAL_VERSION}"]) {
+                        script {
+                            csc.build_integrationtests_conda("main")
+                        }
+                    }
+                }
+            }//Create Release
+            stage("Create Conda dev package") {
+                when {
+                    not {
+                        buildingTag()
+                    }
+                }
+                steps {
+                    withEnv(["WHOME=${env.WORKSPACE}"]) {
+                        script {
+                            csc.build_integrationtests_conda("dev")
+                        }
+                    }
+                }
+            }//Create Dev
+            stage("Push Conda Release package") {
+                when {
+                    buildingTag()
+                    not {
+                        tag pattern: "^v\\d\\.\\d\\.\\d\\.rc\\.\\d\$", comparator: "REGEXP"
+                    }
+                }
+                steps {
+                    withCredentials([usernamePassword(credentialsId: 'CondaForge', passwordVariable: 'anaconda_pass', usernameVariable: 'anaconda_user')]) {
+                        withEnv(["WHOME=${env.WORKSPACE}"]) {
+                            sh """
+                            source /home/saluser/miniconda3/bin/activate
+                            anaconda login --user ${anaconda_user} --password ${anaconda_pass}
+                            """
+                            script {
+                                csc.upload_conda("ts-integrationtests","main","linux-64")
```

Comment:
Should make ts-integrationtests a local variable since it is used more than once. Will also help with the comments about email and Slack ID.